packages/react/src/icon/__snapshots__/Icon.test.tsx.snap

```diff
@@ -167,6 +167,46 @@ exports[`Icon AvatarIcon renders correctly 1`] = `
 </div>
 `;
 
+exports[`Icon ChartBarIcon renders correctly 1`] = `
```

Comment:
Let's merge in https://github.com/steelthreads/agds-next/pull/927 and once that's done we can rebase this PR. That way, this PR will only contain work related to form changes.